eisbuk-admin/package.json

```diff
@@ -16,11 +16,19 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "@types/i18next": "^13.0.0",
+    "@types/i18next-browser-languagedetector": "^3.0.0",
+    "@types/jest": "^24.0.0",
+    "@types/node": "^12.0.0",
+    "@types/react": "^16.9.0",
+    "@types/react-dom": "^16.9.0",
+    "@types/react-i18next": "^8.1.0",
```

Comment:
Please watch out when adding types to use `yarn add -D @types/...` to avoid conflict between `dependencies` and `devDependencies` like we have here. This might actually be an issue with git merging the files wrongly (as I've resolved similar issue a few commits back), but I see that i18next related types are also in `dependencies`. I will take care of this, just please keep this in mind.

Reply:
Yeah, I'll keep that in mind. I'm new to the whole definitely typed thing and typescript in general so I do appreciate all the tips you could give out!